gallery/src/styles/NxPre/NxPreExample.html

```diff
@@ -0,0 +1,15 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+<pre class="nx-pre">
+A line of text.
+	Another line, indented with a tab
+Some text followed by spaces.     Some text followed by tabs.		Some text followed by two newlines
```

Comment:
Minor but I interpret "followed by two newlines" as meaning two empty lines before the _fin_, I only see one.

Reply:
> Minor but I interpret "followed by two newlines" as meaning two empty lines before the fin, I only see one.

A newline is the character that ends a line.  When two newlines follow a string of letters, the result is that the first new line moves the cursor to the next line, and then the second newline moves the cursor down once again, resulting in a single blank line above the cursor.